cpp/ps0004/p4Solution.cpp

```diff
@@ -1,22 +1,25 @@
 #include<iostream>
-using namespace std;
 
+using namespace std;
 
 int main() {
+	// prompting user for number 1.5/1.5
 	float num;
 	char character;
 	cout << "Enter the ascii code[32 : 126] to get its corresponding character: ";
 	cin >> num;
+
+	// checking invalidaty of user's input 4/4
 	if (num == 0)
 		cout << "Invalid character/s or Zero value";
 	else if (num != int(num))
 		cout << "In valid: a floating-point number\n";
-	else if (num < 0 or num >126 or !(num >= 32 and num <= 126))
+	else if (!(num >= 32 && num <= 126))
 		cout << "Invalid: Out of range\n";
 	else {
 		character = static_cast <char>(num);
 		cout << "The Character of Ascii code: " << num << " is " << character << endl;
 	}
 	return 0;
-
-}
\ No newline at end of file
+}
+// great work 😎 8.5/10 ⭐

```

Comment:
- [x] program is in the correct path
- [ ] no syntax errors, bugs or warnings while compilation and execution
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code
- [ ] main returns non-zero/zero number according to the case
+0.5 bonus mark for your effort

you can check the [problem solution](https://onlinegdb.com/9x6wLTlkK) for more